packages/fx-core/src/plugins/resource/aad/utils/telemetry.ts

```diff
@@ -28,18 +28,25 @@ export class TelemetryUtils {
   }
 
   public static sendException(
-    error: Error,
+    eventName: string,
+    errorCode: string,
+    errorType: string,
+    errorMessage: string,
     properties?: { [key: string]: string },
     measurements?: { [key: string]: number }
-  ) {
+  ): void {
     if (!properties) {
       properties = {};
     }
-    properties[Constants.telemetryComponent] = Plugins.pluginNameComplex;
-    TelemetryUtils.ctx.telemetryReporter?.sendTelemetryException(
-      error,
+    
+    properties[Telemetry.component] = Plugins.pluginNameComplex;
+    properties[Telemetry.errorCode] = errorCode;
+    properties[Telemetry.errorType] = errorType;
+    properties[Telemetry.errorMessage] = errorMessage;
+    TelemetryUtils.ctx.telemetryReporter?.sendTelemetryEvent(
```

Comment:
You should call telemetryReporter.sendTelemetryErrorEvent() API here.

Reply:
Fixed. Thanks!